diff --git a/node_modules/react-apollo-hooks/es/useQuery.js b/node_modules/react-apollo-hooks/es/useQuery.js
index bfb6ae3..c481ff6
--- a/node_modules/react-apollo-hooks/es/useQuery.js
+++ b/node_modules/react-apollo-hooks/es/useQuery.js
@@ -94,6 +94,7 @@ export function useQuery(query, _temp) {
   }, [shouldSkip, observableQuery]);
   ensureSupportedFetchPolicy(suspend, fetchPolicy);
   var helpers = {
+    subscribeToMore: observableQuery.subscribeToMore.bind(observableQuery),
     fetchMore: observableQuery.fetchMore.bind(observableQuery),
     refetch: observableQuery.refetch.bind(observableQuery),
     startPolling: observableQuery.startPolling.bind(observableQuery),
diff --git a/node_modules/react-apollo-hooks/lib/useQuery.d.ts b/node_modules/react-apollo-hooks/lib/useQuery.d.ts
index bc79c75..0cb9cc3
--- a/node_modules/react-apollo-hooks/lib/useQuery.d.ts
+++ b/node_modules/react-apollo-hooks/lib/useQuery.d.ts
@@ -12,7 +12,7 @@ export interface QueryHookOptions<TVariables> extends Omit<QueryOptions<TVariabl
     skip?: boolean;
     suspend?: boolean;
 }
-export interface QueryHookResult<TData, TVariables> extends QueryHookState<TData>, Pick<ObservableQuery<TData, TVariables>, 'refetch' | 'startPolling' | 'stopPolling' | 'updateQuery'> {
+export interface QueryHookResult<TData, TVariables> extends QueryHookState<TData>, Pick<ObservableQuery<TData, TVariables>, 'refetch' | 'startPolling' | 'stopPolling' | 'updateQuery' | 'subscribeToMore'> {
     fetchMore<K extends keyof TVariables>(fetchMoreOptions: FetchMoreQueryOptions<TVariables, K> & FetchMoreOptions<TData, TVariables>): Promise<ApolloQueryResult<TData>>;
 }
 export declare function useQuery<TData = any, TVariables = OperationVariables>(query: DocumentNode, { ssr, skip, suspend, pollInterval, notifyOnNetworkStatusChange, context, metadata, variables, fetchPolicy: actualCachePolicy, errorPolicy, fetchResults, }?: QueryHookOptions<TVariables>): QueryHookResult<TData, TVariables>;
diff --git a/node_modules/react-apollo-hooks/lib/useQuery.js b/node_modules/react-apollo-hooks/lib/useQuery.js
index 4a4fcad..3658290
--- a/node_modules/react-apollo-hooks/lib/useQuery.js
+++ b/node_modules/react-apollo-hooks/lib/useQuery.js
@@ -108,6 +108,7 @@ function useQuery(query, _temp) {
   }, [shouldSkip, observableQuery]);
   ensureSupportedFetchPolicy(suspend, fetchPolicy);
   var helpers = {
+    subscribeToMore: observableQuery.subscribeToMore.bind(observableQuery),
     fetchMore: observableQuery.fetchMore.bind(observableQuery),
     refetch: observableQuery.refetch.bind(observableQuery),
     startPolling: observableQuery.startPolling.bind(observableQuery),
